app/org/maproulette/framework/controller/TaskController.scala

```diff
@@ -122,6 +122,43 @@ class TaskController @Inject() (
     }
   }
 
+  /**
+    * Gets all the task markers within a bounding box
+    *
+    * @param left   The minimum latitude for the bounding box
+    * @param bottom The minimum longitude for the bounding box
+    * @param right  The maximum latitude for the bounding box
+    * @param top    The maximum longitude for the bounding box
+    * @param limit  Limit for the number of returned tasks
```

Comment:
The same mixup is also reflected in this doc comment

Reply:
Thanks for catching this too!